manual/dmc.tex

```diff
@@ -99,7 +99,7 @@ \section{Diffusion Monte Carlo}
 \item \texttt{substeps}.  Same as BranchInterval.
 
 
-\item \texttt{nonlocalmoves}. DMC driver for running Hamiltonians with non-local moves.  An typical usage is to simulate Hamiltonians with non-local psuedopotentials with T-Moves.  Setting this equal to false will impose the locality approximation.
+\item \texttt{nonlocalmoves}. DMC driver for running Hamiltonians with non-local moves.  An typical usage is to simulate Hamiltonians with non-local psuedopotentials with T-Moves.  Setting this equal to false will impose the locality approximation. `yes/v0' impmements the algorithm in the 2006 paper~\cite{Casula2006} and `v1' impmements the v1 algorithm in the 2010 paper~\cite{Casula2010}.
```

Comment:
spelling of 'implements'

Reply:
Add "The v1 algorithm is size-consistent and an important advance over the previous v0 non-size consistent algorithm. Investigating the importance of size consistency is highly recommended."